[debug] [2025-09-17T02:36:01.071Z] ----------------------------------------------------------------------
[debug] [2025-09-17T02:36:01.073Z] Command:       /Users/goodluck/.nvm/versions/node/v22.18.0/bin/node /Users/goodluck/.nvm/versions/node/v22.18.0/bin/firebase init
[debug] [2025-09-17T02:36:01.073Z] CLI Version:   14.16.0
[debug] [2025-09-17T02:36:01.073Z] Platform:      darwin
[debug] [2025-09-17T02:36:01.073Z] Node Version:  v22.18.0
[debug] [2025-09-17T02:36:01.074Z] Time:          Wed Sep 17 2025 09:06:01 GMT+0630 (Myanmar Time)
[debug] [2025-09-17T02:36:01.074Z] ----------------------------------------------------------------------
[debug] 
[debug] [2025-09-17T02:36:01.077Z] > command requires scopes: ["email","openid","https://www.googleapis.com/auth/cloudplatformprojects.readonly","https://www.googleapis.com/auth/firebase","https://www.googleapis.com/auth/cloud-platform"]
[debug] [2025-09-17T02:36:01.077Z] > authorizing via signed-in user (soehtuncu@gmail.com)
[info] 
     ######## #### ########  ######## ########     ###     ######  ########
     ##        ##  ##     ## ##       ##     ##  ##   ##  ##       ##
     ######    ##  ########  ######   ########  #########  ######  ######
     ##        ##  ##    ##  ##       ##     ## ##     ##       ## ##
     ##       #### ##     ## ######## ########  ##     ##  ######  ########

You're about to initialize a Firebase project in this directory:

  /Users/goodluck/Desktop/99/test/vue/real-time-chat

[debug] [2025-09-17T02:36:51.316Z] ExitPromptError: User force closed the prompt with SIGINT
    at Interface.sigint (/Users/goodluck/.nvm/versions/node/v22.18.0/lib/node_modules/firebase-tools/node_modules/@inquirer/core/dist/commonjs/lib/create-prompt.js:101:37)
    at Interface.emit (node:events:518:28)
    at Interface.emit (node:domain:489:12)
    at [_ttyWrite] [as _ttyWrite] (node:internal/readline/interface:1132:18)
    at ReadStream.onkeypress (node:internal/readline/interface:271:20)
    at ReadStream.emit (node:events:530:35)
    at ReadStream.emit (node:domain:489:12)
    at emitKeys (node:internal/readline/utils:370:14)
    at emitKeys.next (<anonymous>)
    at ReadStream.onData (node:internal/readline/emitKeypressEvents:64:36)
[error] 
[error] Error: An unexpected error has occurred.
